demo/admin/src/news/Form.tsx

```diff
@@ -18,11 +18,13 @@ import { ArrowLeft } from "@comet/admin-icons";
 import { EditPageLayout } from "@comet/cms-admin";
 import { IconButton } from "@mui/material";
 import { useContentScope } from "@src/common/ContentScopeProvider";
-import { GQLMutationcreateNewsArgs, GQLMutationupdateNewsArgs, GQLNewsInput, GQLNewsQuery, GQLNewsQueryVariables } from "@src/graphql.generated";
+import { GQLMutationcreateNewsArgs, GQLMutationupdateNewsArgs, GQLNewsInput } from "@src/graphql.generated";
```

Comment:
input types and enums stay in central file (as they are the same across the whole application)